.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+<!--Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.-->
+
+Fixes # (issue)<!--Add issue no. which you solved at place of issue-->
```

Comment:
added Addresses # (issue) to mention that it fixes the issue partially.

Reply:
> added Addresses # (issue) to mention that it fixes the issue partially.


do you want me to write `Addresses # (issue)` instead of `Fixes # (issue)`

Reply:
Write both the options (fixes and address) and comment them both. 

<!-- Fixes # (issue)  - use this if your pull request solves the issue completely-->
<!-- Addresses # (issue)  - use this if your pull request solves only a part of the issue-->